torch/distributed/_shard/sharded_tensor/api.py

```diff
@@ -425,6 +425,26 @@ def cpu(
         )
         return st_cpu
 
+    def cuda(
+        self,
+        device=None,
+        non_blocking=False,
+        memory_format=torch.preserve_format,
+        process_group=None
+    ) -> ShardedTensor:
+        """
+        Returns a copy of this object in CUDA memory.
+
+        If this ShardedTensor is already on CPU memory, then no copy is
```

Comment:
So we'd return the CPU sharded tensor as-is, but this seems contradictory to `.cuda()` call?

Reply:
sorry forget to update this call to the correct semantic, updating now.